vlan_assignments.go

```diff
@@ -0,0 +1,174 @@
+package packngo
+
+import (
+	"path"
+)
+
+const (
+	portVLANAssignmentsPath      = "vlan-assignments"
+	portVLANAssignmentsBatchPath = "batches"
+)
+
+type vlanAssignmentsRoot struct {
+	VLANAssignments []VLANAssignment `json:"vlan_assignments"`
+	Meta            meta             `json:"meta"`
+}
+
+type vlanAssignmentBatchesRoot struct {
+	VLANAssignmentBatches []VLANAssignmentBatch `json:"batches"`
+	Meta                  meta                  `json:"meta"`
+}
+
+// VLANAssignmentService handles operations on a VLANAssignment
+type VLANAssignmentService interface {
+	Get(string, string, *GetOptions) (*VLANAssignment, *Response, error)
+	List(string, *ListOptions) ([]VLANAssignment, *Response, error)
+}
+
+type VLANAssignmentBatchService interface {
```

Comment:
Let me know if you think it makes more sense to merge these services into one. 
I think it may be strange to offer a Create that is `CreateBatch` with no single `Create`.  I think I prefer what we have here to the following, but I could be convinced otherwise:

```go
type VLANAssignmentService interface {
	Get(string, string, *GetOptions) (*VLANAssignment, *Response, error)
	List(string, *ListOptions) ([]VLANAssignment, *Response, error)

	GetBatch(string, string, *GetOptions) (*VLANAssignmentBatch, *Response, error)
	ListBatches(string, *ListOptions) ([]VLANAssignmentBatch, *Response, error)
	CreateBatch(string, *VLANAssignmentBatchCreateRequest, *GetOptions) (*VLANAssignmentBatch, *Response, error)
}
```